copis/gui/main_frame.py

```diff
@@ -58,13 +57,16 @@ class MainWindow(wx.Frame):
 
     _FILE_DIALOG_WILDCARD = 'COPIS Files (*.copis)|*.copis|All Files (*.*)|*.*'
 
-    def __init__(self, *args, **kwargs) -> None:
-        """Inits MainWindow with constructors."""
-        super(MainWindow, self).__init__(*args, **kwargs)
+    def __init__(self, chamber_dimensions, *args, **kwargs) -> None:
+        """Initializes MainWindow with constructors"""
```

Comment:
Let's keep it as "Inits" for now, because we use that almost everywhere.

Reply:
I've fixed it everywhere.